pkg/config/config.go

```diff
@@ -170,6 +170,19 @@ func init() {
 	Datadog.BindEnv("bosh_id")
 	Datadog.BindEnv("histogram_aggregates")
 	Datadog.BindEnv("histogram_percentiles")
+
+	// Logs
+	BindEnvAndSetDefault("log_enabled", false)
+	BindEnvAndSetDefault("logset", "")
+	BindEnvAndSetDefault("log_dd_url", "intake.logs.datadoghq.com")
+	BindEnvAndSetDefault("log_dd_port", 10516)
+	BindEnvAndSetDefault("run_path", "/opt/datadog-agent/run")
```

Comment:
I think we should go with the alternative you mentioned: `defaultJMXPipePath` is not used anymore so let's remove it, instead we should have a `defaultRunPath` that points to `/opt/datadog-agent/run/` on unix by default (not sure about Windows, feel free to add a `FIXME` for windows).

The logs agent can then use that path.